config/initializers/wifi_density.rb

```diff
@@ -0,0 +1,2 @@
+config_file_location = Rails.root.join('config/wifi_density.yml')
+WIFI_DENSITY = File.exists?(config_file_location) ? YAML.load_file(config_file_location) : {}
```

Comment:
Could use `Rails.application.config_for(:wifi_density)`

Reply:
This would also make it a little less annoying to override in a view test.